src/components/TaskListItem.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <li class="task">
+    <input type="checkbox" v-model="instanceStatus">
+
+    <label>
+      ( {{ loggedSize }} /<span contenteditable @keydown.enter.prevent.stop="updateEstimatedSize">
+      {{ estimatedSize }}
+      </span>)
+      <span contenteditable @keydown.enter.prevent.stop="updateDescription">
+        {{ description }}</span>
+      <a href="#" @click.prevent="deleteTask">[x]</a>
+    </label>
+  </li>
+</template>
+
+<script>
+export default {
+  name: 'task-list-item',
+
+  props: {
+    taskId: Number,
+    description: String,
+    estimatedSize: Number,
+    type: String,
+    loggedSize: Number,
+    status: Boolean
+  },
+  data () {
+    return {
+      instanceStatus: this.status
+    }
+  },
+  watch: {
+    instanceStatus: {
+      handler: 'onInstanceStatusChanged',
+      immediate: true,
+      deep: false
+    }
+  },
+  methods: {
+    onInstanceStatusChanged (newVal) {
+      this.$emit('updateTask', { id: this.taskId, status: newVal })
```

Comment:
Not implemeted yet?

Reply:
Yes. I bring this code from typescript branch.
Do you want erase this?

Reply:
Nope. I hope to check. It's ok to leave it